scoap3/modules/workflows/workflows/articles_upload.py

```diff
@@ -238,6 +241,27 @@ def build_files_data(obj, eng):
              'name': '{0}.xml'.format(doi),
              'filetype': 'xml'}
         ]
+    elif method == 'scoap3_push':
+        files = []
+        for document in obj.data.get('documents', ()):
+            if 'xml' in document['key']:
+                filetype = 'xml'
+            elif 'pdfa' in document['key']:
+                filetype = 'pdfa'
+            elif 'pdf' in document['key']:
+                filetype = 'pdf'
+            else:
+                filetype = 'unknown'
```

Comment:
What about a more pythonic statement like :point_down: ? 
```python
known_extensions = ('xml', 'pdf', 'pdf')
extension = document['key'] if document['key'] in known_extensions else 'unknown'
```

Reply:
The reason for not using this is that I'm checking for a substring, not equation.